cabot/metricsapp/api/base.py

```diff
@@ -0,0 +1,123 @@
+from celery.utils.log import get_task_logger
+import json
+import time
+from cabot.cabotapp.models import Service, StatusCheckResult
+
+logger = get_task_logger(__name__)
+
+
+def run_metrics_check(check):
```

Comment:
minor style recommendation (which I picked up from c programming days)

I prefer to define all the static / private functions in the beginning of the file and the public functions at the end. Makes for good readability.

```
def _boo(a, b):

def _foo():

def main_func():
   a = _foo()
   return _boo(a, 5)
```